src/components/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import gql from 'graphql-tag';
+import { useMutation } from '@apollo/react-hooks';
+
+
+const SIGN_UP = gql`
```

Comment:
Would be preferable to keep your constant mutations query somewhere else than in the component definition, like an external module that contains your query as a dictionary.